docs/literate/src/files/adding_nonconservative_equation.jl

```diff
@@ -176,6 +176,76 @@ error_1 / error_2
 # For non-trivial boundary conditions involving non-conservative terms,
 # please refer to the section on [Other available example elixirs with non-trivial BC](https://trixi-framework.github.io/Trixi.jl/stable/tutorials/non_periodic_boundaries/#Other-available-example-elixirs-with-non-trivial-BC).
 
+# Some additional routines need modified versions to avoid adding dissipation to the
```

Comment:
Can we move this part below the section "Summary of the code" and turn it into a new section, e.g., "Additional modifications"?

Reply:
Sure, that sounds like a good idea.

Reply:
Done in 5b1442fd13ce461d9bf6badc063dfd1a50bd92a8